### ISSUE TITLE
Vue3 imports in JS file - Cannot read property 'type' of undefined

### ISSUE THREAD

**Issue by andyghiuta** [vuejs/eslint-plugin-vue#1103]
**Tell us about your environment**	
* **ESLint version:**	v6.8.0
* **eslint-plugin-vue version:**	7.0.0-alpha.0
* **Node version:**	10.18.1

**Please show your full configuration:**

```js
{
  root: true,
  env: {
    node: true,
  },
  extends: [
    'plugin:vue/vue3-recommended',
    '@vue/airbnb',
  ],
  parserOptions: {
    parser: 'babel-eslint',
  },
}
```

**What did you do?**	

```js
import { computed, ref, onMounted } from 'vue';

const useSomething = () => {
  const r = ref(0);
  const c = computed(r, () => null);
  onMounted(() => {
    console.log(1);
  });
};

```

**What did you expect to happen?**
The linting to work.

**What actually happened?**
Linting stops the moment any of `ref`, `onMounted`, `onUpdated` (probably others too) are imported.

```
 ERROR  TypeError: Cannot read property 'type' of undefined
TypeError: Cannot read property 'type' of undefined
    at ReferenceTracker._iteratePropertyReferences (D:\path\to\project\node_modules\eslint-utils\index.js:1552:20)
    at _iteratePropertyReferences.next (<anonymous>)
    at ReferenceTracker._iterateVariableReferences (D:\path\to\project\node_modules\eslint-utils\index.js:1530:29)
    at _iterateVariableReferences.next (<anonymous>)
    at ReferenceTracker._iterateImportReferences (D:\path\to\project\node_modules\eslint-utils\index.js:1694:25)
    at _iterateImportReferences.next (<anonymous>)
    at ReferenceTracker.iterateEsmReferences (D:\path\to\project\node_modules\eslint-utils\index.js:1490:25)
    at iterateEsmReferences.next (<anonymous>)
    at Program (D:\path\to\project\node_modules\eslint-plugin-vue\lib\rules\no-lifecycle-after-await.js:56:29)
    at listeners.(anonymous function).forEach.listener (D:\path\to\project\node_modules\eslint\lib\util\safe-emitter.js:47:58)
    at Array.forEach (<anonymous>)
    at Object.emit (D:\path\to\project\node_modules\eslint\lib\util\safe-emitter.js:47:38)
    at NodeEventGenerator.applySelector (D:\path\to\project\node_modules\eslint\lib\util\node-event-generator.js:251:26)
    at NodeEventGenerator.applySelectors (D:\path\to\project\node_modules\eslint\lib\util\node-event-generator.js:280:22)
    at NodeEventGenerator.enterNode (D:\path\to\project\node_modules\eslint\lib\util\node-event-generator.js:294:14)
    at CodePathAnalyzer.enterNode (D:\path\to\project\node_modules\eslint\lib\code-path-analysis\code-path-analyzer.js:608:23)
```

**Comment by ota-meshi** [comment]
Thank you for this issue.

However I can't reproduce this issue. Could you share a minimal repository that can reproduce this issue?

**Comment by andyghiuta** [comment]
Thanks for pointing me in that direction. I should have tried that first...
I found the culprit. I had eslint installed globally (v6.8.0) not locally. Here's a repo where this issue can be observed: https://github.com/andyghiuta/eslint-plugin-vue-1103/

I broke it with this commit: https://github.com/andyghiuta/eslint-plugin-vue-1103/commit/6e0ae65ddc63e433373cd48a82e41f958bb2c5a0

**Comment by ota-meshi** [comment]
I was able to reproduce the problem using `eslint@4`. You may have used `eslint@4`?

**Comment by andyghiuta** [comment]
No. I had 6.8.0 globally, but anyway, I think it can be closed, since having the 6.7.2 version locally works

**Comment by ota-meshi** [comment]
Hmm, I don't know the cause. But you seem to be able to solve this problem by using a locally installed eslint, so I close this issue. If you cannot solve this problem, reopen this issue.
